diff --git a/libfprint/sigfm/sigfm.cpp b/libfprint/sigfm/sigfm.cpp
index 0781299..a4ba730 100644
--- a/libfprint/sigfm/sigfm.cpp
+++ b/libfprint/sigfm/sigfm.cpp
@@ -123,7 +123,7 @@ SigfmImgInfo* sigfm_extract(const SigfmPix* pix, int width, int height)
     std::vector<cv::KeyPoint> pts;
 
     cv::Mat descs;
-    cv::SIFT::create()->detectAndCompute(img, roi, pts, descs);
+    cv::SIFT::create(64)->detectAndCompute(img, roi, pts, descs);
 
     auto* info = new SigfmImgInfo{pts, descs};
     return info;
